From 38d3c1f4b7a109a8f80dd656ab4954f5508814c8 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Mon, 18 Dec 2023 19:44:37 +0100
Subject: [PATCH 0773/1141] dt-bindings: spi: stm32: add 'power-domains'
 property

STM32 SPI may be in a power domain which is the case for the STM32MP25
based boards. Allow a single 'power-domains' entry for STM32 SPI.

Fix dtbs_check warning:
Unevaluated properties are not allowed ('power-domains' was unexpected)

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ib8c58e11b4181b9632990c7bf5ed1ad4eb5d6fe3
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/347861
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 Documentation/devicetree/bindings/spi/st,stm32-spi.yaml | 3 +++
 1 file changed, 3 insertions(+)

--- a/Documentation/devicetree/bindings/spi/st,stm32-spi.yaml
+++ b/Documentation/devicetree/bindings/spi/st,stm32-spi.yaml
@@ -52,6 +52,9 @@ properties:
   resets:
     maxItems: 1
 
+  power-domains:
+    maxItems: 1
+
   dmas:
     description: |
       DMA specifiers for tx and rx dma. DMA fifo mode must be used. See
